pkgs/applications/networking/misc/zammad/update.sh

```diff
@@ -8,7 +8,8 @@ if [ "$#" -gt 2 ] || [[ "$1" == -* ]]; then
   exit 1
 fi
 
-VERSION=$(curl -s https://ftp.zammad.com/ | grep -v latest | grep tar.gz | sed "s/<a href=\".*\">zammad-//" | sort -h | tail -n 1 | awk '{print $1}' | sed 's/.tar.gz<\/a>//')
+#VERSION=$(curl -s https://ftp.zammad.com/ | grep -v latest | grep tar.gz | sed "s/<a href=\".*\">zammad-//" | sort -h | tail -n 1 | awk '{print $1}' | sed 's/.tar.gz<\/a>//')
```

Comment:
VERSIO=$(xidel -s https://ftp.zammad.com/ --extract "//a" | grep -E "zammad-[0-9.]*.tar.gz" | sort --version-sort | tail -n 1)

Reply:
oops, did'nt mean to commit this :P

Reply:
Ah, thought you were looking for an alternative to what you have :D

Reply:
that too...